utils/bitfield/src/rleplus.rs

```diff
@@ -186,12 +183,7 @@ pub fn decode(enc: &BitVec<Lsb0, u8>) -> Result<BitVec<Lsb0, u8>, &'static str>
                     Some(true) => {
```

Comment:
this was probably something that was already written before in the past but I think we could put the get(i) on an if statement and then use ok_or_else for the get(i+1) would just reduce the amount of matching that we would have to do

Reply:
I'm sorry I'm having trouble thinking through what you mean by this, could you maybe push a commit to some branch with this change or explain it differently? It was written in the past yes, but happy to clean up anything that can be (not a ton of thought was put into cleaning this up, was more for functionality)

Reply:
Woops sorry about that I think my explanation was at fault there but I was thinking maybe this but now I don't know how much of an improvement it adds

https://github.com/ChainSafe/forest/commit/ff62f1bfca4d45027698f887df682c48761e9f0e

Reply:
I actually do like this, not really a functional benefit but I think it's cleaner! Generally I would do in a seperate PR but I'll make the change here